internal/cmd/base/create_test.go

```diff
@@ -0,0 +1,155 @@
+package base_test
+
+import (
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/hetznercloud/cli/internal/cli"
+	"github.com/hetznercloud/cli/internal/cmd/base"
+	"github.com/hetznercloud/cli/internal/cmd/util"
+	"github.com/hetznercloud/cli/internal/hcapi2"
+	"github.com/hetznercloud/cli/internal/state"
+	"github.com/hetznercloud/cli/internal/testutil"
+)
+
+type fakeResource struct {
+	ID   int    `json:"id"`
+	Name string `json:"name"`
+}
+
+var commandCalled bool
```

Comment:
Is this really needed, can't you check the returned value of the cmd?

Reply:
Sometimes the output is empty (for example in `TestCreateQuiet`)